src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double RADIUS = 10;
+    private static final int MAX_RANDOM_NUMBER = 10;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
These fields should not be the `static`
```suggestion
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```